packages/core/types/index.js

```diff
@@ -339,16 +348,25 @@ type ResolveConfigFn = (
 export type Validator = {|
   validate({
     asset: Asset,
+    localRequire: LocalRequire,
     resolveConfig: ResolveConfigFn, // This is a temporary function and should be replaced with something cacheable
     options: ParcelOptions
   }): Async<void>
 |};
 
+export type LocalRequire = (
+  name: string,
+  path: FilePath,
+  triedInstall?: boolean
+  // $FlowFixMe
+) => Promise<any>;
+
 export type Transformer = {
   getConfig?: ({
     asset: MutableAsset,
     resolve: ResolveFn,
-    options: ParcelOptions
+    options: ParcelOptions,
+    localRequire: LocalRequire
```

Comment:
ah interesting. I was working on moving this into a package manager that would get passed around. seems like an ok temporary solution though